gulp/gulp-config.js

```diff
@@ -65,6 +65,7 @@ module.exports = {
       './node_modules/stickyfill/index.js',
       './node_modules/jquery-ui-bundle/jquery-ui.min.css',
       './node_modules/jquery-ui-bundle/jquery-ui.min.js',
+      './node_modules/bootstrap/dist/js/bootstrap.bundle.min.js',
```

Comment:
is this a css file?

Reply:
You mean './node_modules/jquery-ui-bundle/**jquery-ui.min.css**' ?

Config contains all the external plugin files both JS and CSS

Reply:
i'm meaning why is .js files included in the css array.

Reply:
It is not, Github change view is a bit confusing. If you see the complete file. The only transferToBuild contains all types of the files because this is just an array of files we want to transfer from Node Modules to SWE.

https://github.com/qld-gov-au/qg-web-template/blob/release-v4.0.4/gulp/gulp-config.js

`   js: ['generate-id.js',
      'jquery.resize-events.js',
      'butterfly/jquery.history.js',
      'butterfly/jquery.butterfly.js',
      'bootstrap/dist/js/bootstrap.bundle.min.js',
    ],
    css: [
      'butterfly/butterfly.css',
    ],
    transferToBuild: [
      './node_modules/nodep-date-input-polyfill/nodep-date-input-polyfill.dist.js',
      './node_modules/stickyfill/index.js',
      './node_modules/jquery-ui-bundle/jquery-ui.min.css',
      './node_modules/jquery-ui-bundle/jquery-ui.min.js',
      './node_modules/bootstrap/dist/js/bootstrap.bundle.min.js',
      './node_modules/jquery-ui-bundle/images/**',
    ],`